From: uazo <uazo@users.noreply.github.com>
Date: Wed, 19 Jul 2023 09:32:36 +0000
Subject: Compress libchrome to free up some space

upstream removed ModernLinker support for developer build (apk releases).
see https://bugs.chromium.org/p/chromium/issues/detail?id=1383210
---
 chrome/android/chrome_public_apk_tmpl.gni | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/android/chrome_public_apk_tmpl.gni b/chrome/android/chrome_public_apk_tmpl.gni
--- a/chrome/android/chrome_public_apk_tmpl.gni
+++ b/chrome/android/chrome_public_apk_tmpl.gni
@@ -632,6 +632,7 @@ template("chrome_common_apk_or_module_tmpl") {
           [ "//components/crash/core/app:chrome_crashpad_handler_named_as_so" ]
       loadable_modules += [ "$root_out_dir/libchrome_crashpad_handler.so" ]
       library_always_compress += [ "libchrome_crashpad_handler.so" ]
+      library_always_compress += [ "libchrome.so" ]
     } else if (!_is_trichrome) {
       # Crashpad trampoline lives in TrichromeLibrary.apk.
       # https://chromium.googlesource.com/chromium/src/+/main/docs/android_native_libraries.md#Crashpad-Packaging
--
2.25.1
